core/filter_api.php

```diff
@@ -1492,6 +1492,7 @@ function filter_get_bug_rows_query_clauses( array $p_filter, $p_project_id = nul
 		$t_private_and_public_project_ids = array();
 		$t_limited_projects = array();
 
+		project_cache_array_rows( $t_project_ids );
```

Comment:
It would be good to add the comments about why this is called, similar to the comment in the commit message.